### ISSUE TITLE
Changing vo does not trigger change event for window-id property

### ISSUE THREAD

**Issue by tsl0922** [mpv-player/mpv#13495]
### Important Information

- mpv version: v0.37.0-316-g98005003
- Windows Version: Windows 10 22H2
- Source of the mpv binary: shinchiro

### Reproduction steps

```lua
local msg = require('mp.msg')

mp.observe_property('window-id', 'native', function(name, val)
	msg.info(name, val)
end)
```

Run `set vo gpu-next` in mpv console

### Expected behavior

The lua code should print the new window-id after vo changed.

### Actual behavior

Nothing printed after vo changed.

**Comment by Dudemanguy** [comment]
Yeah looks like this was forgotten in the event property updaters. This will probably fix it. Will confirm in a bit.

```
diff --git a/player/command.c b/player/command.c
index 66b808e438..8d40df98ab 100644
--- a/player/command.c
+++ b/player/command.c
@@ -4075,8 +4075,8 @@ static const char *const *const mp_event_property_change[] = {
     E(MPV_EVENT_VIDEO_RECONFIG, "video-out-params", "video-params",
       "video-format", "video-codec", "video-bitrate", "dwidth", "dheight",
       "width", "height", "container-fps", "aspect", "aspect-name", "vo-configured", "current-vo",
-      "video-dec-params", "osd-dimensions",
-      "hwdec", "hwdec-current", "hwdec-interop"),
+      "video-dec-params", "osd-dimensions", "hwdec", "hwdec-current", "hwdec-interop"
+      "window-id"),
     E(MPV_EVENT_AUDIO_RECONFIG, "audio-format", "audio-codec", "audio-bitrate",
       "samplerate", "channels", "audio", "volume", "volume-gain", "mute",
       "current-ao", "audio-codec-name", "audio-params",
```

**Comment by Dudemanguy** [comment]
@tsl0922: Could you verify if the build linked [here](https://github.com/mpv-player/mpv/pull/13496#issuecomment-1949793043) works? The window id on x11 on linux actually never changes, so I can't test.

**Comment by tsl0922** [comment]
No, it doesn't.

```
[test] window-id nil
[vo/gpu/d3d11] Using a software adapter
[test] window-id 459574
[vo/gpu-next/d3d11] Using a software adapter
[vo/gpu-next/libplacebo] libplacebo compiled without LittleCMS 2 support!```

**Comment by N-R-K** [comment]
> No, it doesn't.

Because those strings are getting concatenated together due to the missing comma. Does it work with the comma fix? https://github.com/mpv-player/mpv/pull/13496#discussion_r1493246610

**Comment by tsl0922** [comment]
> > No, it doesn't.
> 
> Because those strings are getting concatenated together due to the missing comma. Does it work with the comma fix? [#13496 (comment)](https://github.com/mpv-player/mpv/pull/13496#discussion_r1493246610)

Yes, it works.

**Comment by Dudemanguy** [comment]
Sorry, late at night.
